### PR TITLE
[Feature/54476, 54475, 54481, 54482, 54483, 54484, 54485] iOS: Lithuanian support

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.iOS/Info.plist [hunk 1]
```diff
@@ -104,6 +104,7 @@
 		<string>nn</string>
 		<string>pl</string>
 		<string>so</string>
+		<string>lt</string>
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.iOS/Info.plist:107]: Rename to "Lit". E.g. if they want "Letvia" (lt / let) later on

File: NDB.Covid19/NDB.Covid19.iOS/Views/Initializer/LanguageSelectionViewController.cs [hunk 2]
```diff
@@ -45,73 +47,57 @@ private void InitButtons()
...
-            LocalPreferencesHelper.SetAppLanguage("ar");
-            LocalesService.Initialize();
-            LayoutUtils.OnLayoutDirectionChange();
-            Continue();
+            SetAppLanguageAndContinue("ar");
+        }
+
+        partial void LithuanianButton_TouchUpInside(NSObject sender)
+        {
+            SetAppLanguageAndContinue("lt");
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.iOS/Views/Initializer/LanguageSelectionViewController.cs:80]: lit

File: NDB.Covid19/NDB.Covid19.iOS/Views/Initializer/LanguageSelectionViewController.cs [hunk 3]
```diff
@@ -25,6 +25,7 @@ private void InitButtons()
             StyleUtil.InitButtonStyling(NynorskButton, LanguageSelectionViewModel.LANGUAGE_SELECTION_NN);
             StyleUtil.InitButtonStyling(EnglishButton, LanguageSelectionViewModel.LANGUAGE_SELECTION_EN);
             StyleUtil.InitButtonStyling(ArabicButton, LanguageSelectionViewModel.LANGUAGE_SELECTION_AR);
+            StyleUtil.InitButtonStyling(LithuanianButton, LanguageSelectionViewModel.LANGUAGE_SELECTION_LT);
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.iOS/Views/Initializer/LanguageSelectionViewController.cs:28]: lit

File: NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPageGeneral/SettingsPageGeneralSettingsViewController.cs [hunk 4]
```diff
@@ -62,14 +62,15 @@ void SetupStyling()
...
-            InitLabel(RadioButton3Lbl, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_EN, 16, 28);
-            InitLabel(RadioButton4Lbl, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_PL, 16, 28);
-            InitLabel(RadioButton5Lbl, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_SO, 16, 28);
-            InitLabel(RadioButton6Lbl, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_TI, 16, 28);
-            InitLabel(RadioButton7Lbl, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_AR, 16, 28);
-            InitLabel(RadioButton8Lbl, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_UR, 16, 28);
+            InitLabel(BokmalLabel, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_NB, 16, 28);
+            InitLabel(NynorskLabel, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_NN, 16, 28);
+            InitLabel(EnglishLabel, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_EN, 16, 28);
+            InitLabel(LithuanianLabel, FontType.FontBold, SettingsGeneralViewModel.SETTINGS_GENERAL_LT, 16, 28);
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPageGeneral/SettingsPageGeneralSettingsViewController.cs:68]: lit

File: NDB.Covid19/NDB.Covid19/Configuration/Conf.cs [hunk 5]
```diff
@@ -27,7 +27,7 @@ public class Conf
         //The timeout value takes this into consideration.
         public static int DEFAULT_TIMEOUT_SERVICECALLS_SECONDS => 40;
         public static string DEFAULT_LANGUAGE = "nb"; //In case the device is set to use an unsupported language
-        public static string[] SUPPORTED_LANGUAGES = new string[] { "en", "nb", "nn", "pl", "so", "ti", "ar", "ur" };
+        public static string[] SUPPORTED_LANGUAGES = new string[] { "en", "nb", "nn", "pl", "so", "ti", "ar", "ur", "lt" };
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Configuration/Conf.cs:30]: lit

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 6]
```diff
@@ -100,6 +100,7 @@
   "LANGUAGE_SELECTION_NN": "Norwegian (Nynorsk)",
   "LANGUAGE_SELECTION_EN": "English",
   "LANGUAGE_SELECTION_AR": "Arabic (العربية)",
+  "LANGUAGE_SELECTION_LT": "Lithuanian (translation missing),,
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:103]: a comma too much

File: NDB.Covid19/NDB.Covid19/Locales/lt.json [hunk 7]
```diff
@@ -0,0 +1,349 @@
+﻿{
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/lt.json:1]: Name it "lit.json"

File: NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj [hunk 8]
```diff
@@ -92,6 +92,7 @@
     <None Remove="Locales\ti.json" />
     <None Remove="Locales\ar.json" />
     <None Remove="Locales\ur.json" />
+    <None Remove="Locales\lt.json" />
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj:95]: lit

File: NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj [hunk 9]
```diff
@@ -102,5 +103,6 @@
     <EmbeddedResource Include="Locales\ti.json" />
     <EmbeddedResource Include="Locales\ar.json" />
     <EmbeddedResource Include="Locales\ur.json" />
+    <EmbeddedResource Include="Locales\lt.json" />
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj:106]: lit

File: NDB.Covid19/NDB.Covid19/ViewModels/LanguageSelectionViewModel.cs [hunk 10]
```diff
@@ -8,6 +8,7 @@ public class LanguageSelectionViewModel
         public static string LANGUAGE_SELECTION_NN => "LANGUAGE_SELECTION_NN".Translate();
         public static string LANGUAGE_SELECTION_EN => "LANGUAGE_SELECTION_EN".Translate();
         public static string LANGUAGE_SELECTION_AR => "LANGUAGE_SELECTION_AR".Translate();
+        public static string LANGUAGE_SELECTION_LT => "LANGUAGE_SELECTION_LT".Translate();
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/ViewModels/LanguageSelectionViewModel.cs:11]: lit

File: NDB.Covid19/NDB.Covid19/ViewModels/SettingsGeneralViewModel.cs [hunk 11]
```diff
@@ -23,6 +23,7 @@ public class SettingsGeneralViewModel
         public static string SETTINGS_GENERAL_EN => "SETTINGS_GENERAL_EN".Translate();
         public static string SETTINGS_GENERAL_NN => "SETTINGS_GENERAL_NN".Translate();
         public static string SETTINGS_GENERAL_NB => "SETTINGS_GENERAL_NB".Translate();
+        public static string SETTINGS_GENERAL_LT => "SETTINGS_GENERAL_LT".Translate();
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/ViewModels/SettingsGeneralViewModel.cs:26]: lit

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Info.plist (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Info.plist:107]: Don’t really get it. We’ve used two letter ISO codes for every other language, and Latvian is lv. I also think using something else will cause issues with listing supported languages in app stores.

Comment (Frederik-S-N-dk) [reply NDB.Covid19/NDB.Covid19.iOS/Info.plist:107]: Fair point @jonasofrich 
Don't implement my changes then.

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**